common-crafting.lic

```diff
@@ -181,4 +181,38 @@ module DRCC
       DRC.bput("stow my #{logbook} logbook", 'You put', 'What were you referring to')
     end
   end
+
+  def order_enchant(stock_room, stock_needed, stock_number)
+    stock_needed.times do
+      order_item(stock_room, stock_number)
+      stow_crafting_item(left_hand, @bag, @belt)
```

Comment:
❓ Will `@bag` and `@belt` resolve correctly? I think you'll need to make them function args

Reply:
The resolve for me. stow_crafting_item is a function in common-crafting.

Reply:
I think you've lucked into some state in which those happen to be populated. This is probably due to some nuance of how import works and we shouldn't rely on that. Using function args is safer.